src/opm/output/eclipse/CreateUdqDims.cpp

```diff
@@ -78,21 +133,38 @@ std::size_t entriesPerIGph(const std::vector<int>& inteHead)
 std::vector<int>
 Opm::RestartIO::Helpers::
 createUdqDims(const Schedule&     		sched,
-              const std::size_t       simStep,
-              const std::vector<int>& inteHead)
+              const std::size_t        	lookup_step,
+              const std::vector<int>&   inteHead) 
 {
-    const auto& udqCfg = sched.getUDQConfig(simStep);
-    const auto& udqActive = sched.udqActive(simStep);
-    std::vector<int> udqDims(8);
-
-    udqDims[0] = udqCfg.size();
-    udqDims[1] = entriesPerIUDQ();
-    udqDims[2] = udqActive.IUAD_size();
-    udqDims[3] = entriesPerIUAD();
-    udqDims[4] = entriesPerZUDN();
-    udqDims[5] = entriesPerZUDL();
-    udqDims[6] = entriesPerIGph(inteHead);
-    udqDims[7] = udqActive.IUAP_size();
+    const auto& udqCfg = sched.getUDQConfig(lookup_step);
+    const auto& udqActive = sched.udqActive(lookup_step);
+    std::vector<int> udqDims; 
```

Comment:
Personally I would much prefer if you used the simpler:
```
udqDims[0] = udqCfg.size();
udqDims[1] = enbtriesPerIUDQ();
udqDims[2] = ....
```
The use of `push_back/emplace_back` hints that this is a dynamic datastructure - that is actually not the case; it is rather a very strictly regulated datastructure - dictated by external requirements; that is in my opinion better communicated with the `udqDims[n] = ...` style.

Reply:
The code style you now suggest was the one I used originally, however, you changed it to using the current emplace_back style - I am not sure why you changed style in the first place. I agree that the other style is better since it is less error prone - will change back!

Reply:
🤔 - that was emberrassing, I must have been drinking. Anyway very good if you change it back. Sorry for unclear feedback.